packages/@aws-cdk/aws-neptune-alpha/README.md

```diff
@@ -138,6 +138,17 @@ new neptune.DatabaseCluster(this, 'Cluster', {
 });
 ```
 
+You can also specify `autoMinorVersionUpgrade` to a database instance.
+Even within the same cluster, you can modify the `autoMinorVersionUpgrade` setting on a per-instance basis.
```

Comment:
I created an integration test with a mixed `autoMinorVersionUpgrade` setting for verification, and it is deploying successfully.